### ISSUE TITLE
Eslint 3 support

### ISSUE THREAD

**Issue by SimenB** [Flet/eslint-config-semistandard#11]
Any chance of an update that allows eslint 3?

**Comment by jokeyrhyme** [comment]
We could wait for upstream https://github.com/Flet/eslint-config-semistandard to be bumped to 6.0, but 5.3.2 and newer can already be used with ESLint 3

**Comment by jokeyrhyme** [comment]
May also need to wait for upstream https://github.com/feross/standard to be bumped to 8.0, but it is only a devDependency, so this part should not affect consumers.

**Comment by SimenB** [comment]
@Flet perfect, thanks!

**Comment by SimenB** [comment]
@Flet this was published on the `latest` tag, probably should be on the `beta` tag :smile:

```sh
$ npm show eslint-config-semistandard dist-tags
{ latest: '7.0.0-beta.0' }
```

**Comment by voxpelli** [comment]
Just a note that the upstream projects have now reached stable releases. I guess that means this project can soon get a stable new release as well?

**Comment by Flet** [comment]
indeed, I'll update everything to match the latest standard in the day or so.

**Comment by Flet** [comment]
Bumped up to eslint 3 and the latest `eslint-config-standard`
https://github.com/Flet/eslint-config-semistandard/commit/742c713680e171e7dbeac33cfa2da790af68183b

published as `eslint-config-semistandard@7.0.0`
